.npm/fuel-asm/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "fuel-asm",
+  "version": "0.0.1",
```

Comment:
might be easier if we just keep this version in sync with the rust repo, and add some CI checks to enforce that they always match

Reply:
Yes, of course. The CI automation and publishing steps, which are still _pending_, should take care of this parity.

Reply:
@Voxelot I was naively assuming that the version bumping was somehow an automated thing on the CI _(like we have on the `ts-sdk`_), but I realized it's manually set on `Cargo.toml`:

https://github.com/FuelLabs/fuel-vm/blob/b132c8e03b5bd65afd13d1525189a66b06370360/Cargo.toml#L21

So, I started getting it from there.

https://github.com/FuelLabs/fuel-vm/blob/b132c8e03b5bd65afd13d1525189a66b06370360/wasm_publish.sh#L12